### DISCUSSION TITLE
New Github Default Release/Tag URL is Skipped for Bazel

### DISCUSSION THREAD

**Discussion by cpsauer** [renovatebot/renovate#17496] [Request Help] [answered]
Hello again, wonderful Renovate folks.

GitHub seems to have changed the default URL it exposes for releases/tags. At least for Bazel, Renovate seems to skip these new URLs.

Links from the [release page](https://github.com/libcpr/cpr/releases) are now, e.g., `https://github.com/libcpr/cpr/archive/**refs/tags/**1.9.0.tar.gz`
Instead of `https://github.com/libcpr/cpr/archive/1.9.0.tar.gz` (Added region starred above for emphasis. Actual URL does not contain stars.)

The old links continue to work, both with Renovate and for downloading. But I suspect lots of users will copy over the new link type from GitHub and be surprised that they silently fail to update. (More generally, I still maintain that it might be quite valuable to try to actively notify users when packages they've added are being skipped. Perhaps in a folded section of the dashboard? That'd help people find issues like this.)

Minimal reproduction: https://github.com/cpsauer/renovate-reproduction-new-archive-url
Note the PR to update the entry with the old form URL but not the new form.

The log listed "skipReason": "invalid-value" and "currentValue": "refs", so it looks like it's mis-parsing based on the old url.

Cheers, and thanks for all you do.
Chris

P.S. This is with GitHub app Renovate. AFAIK this is a new URL scheme, as stated above, so I haven't seen it working.
P.P.S. I think I see a reference to this new URL format here https://github.com/renovatebot/renovate/blob/d5a167c5177cbb632308fa0ddb6632e0b4c20614/lib/modules/datasource/github-tags/cache.ts#L15
P.P.P.S Seems like the [Bazel docs](https://docs.renovatebot.com/bazel/) have gotten a fair bit out of date with the functionality. In particular at least zip files and tag functionality are present and super handy, but missing from the docs.

**Comment by rarkins** [comment, marked as answer]
If GitHub or any other third party service changes URL structures, it's not a Renovate bug, so I have converted this to a discussion.

For the new URL schema, please refile this as a feature request, using our feature request issue template. It will be blocked by #9667

For the docs, PRs can be submitted to update https://github.com/renovatebot/renovate/blob/main/docs/usage/bazel.md

**Reply by cpsauer** [reply]
Thanks for replying!

Okay, will do.

[That said, why not just flip the label to feature? Asking as feedback because future folks might find that kinder.]
(Had listed as bug in good faith, thinking it was likely unintended behavior, since there were already references in Renovate's code to the schema, it's not that new, it seemed likely to break things for most new (Bazel?) users, if not me, it was best expressed with a minimal reproduction, and it matched that template somewhat better. Edits away from template and phrasing were intended to be for friendliness and clarity; most template fields were not relevant, so I thought this would let you parse it faster. Feel free to coach me if any of that seems misguided!)

Separately, re docs, it looks like the code's structure has changed considerably since I last contributed, so I'd assessed that I didn't have comparative advantage vs you guys in figuring out the current functionality. Hence the heads up rather than a PR.

Again, @rarkins, thank you for all the wonderful things you do.
